vm/src/vm/runners/cairo_runner.rs

```diff
@@ -78,24 +78,34 @@ impl From<Vec<MaybeRelocatable>> for CairoArg {
 #[derive(Clone, Default, Debug, PartialEq)]
 pub struct RunResources {
     n_steps: usize,
+    consume_steps: bool,
```

Comment:
```suggestion
    should_consume_steps: bool,
```
nit

Reply:
I end up with the n_steps: Option<usize> solution